app.py

```diff
@@ -18,20 +18,23 @@
             "FQDN": "prod.schematic.io",
             "CERTIFICATE_ARN": "arn:aws:acm:us-east-1:878654265857:certificate/d11fba3c-1957-48ba-9be0-8b1f460ee970",
             "TAGS": {"CostCenter": "NO PROGRAM / 000000"},
+            "SCHEMATIC_CONTAINER_LOCATION": "ghcr.io/sage-bionetworks/schematic:v0.1.94-beta",
         }
```

Comment:
Will this deploy to prod on this pr or does it wokr like the previous repo, where pushing to prod branch triggers deploy to prod.

Reply:
> where pushing to prod branch triggers deploy to prod.

That is correct. Only changes deployed to the "prod" branch will be deployed to prod. My thought behind having this configured like this is that ahead of time we could update these values. We can also limit the number of changes that might need to be separate in each long-lived branches to keep them all in sync. Meaning we can always move changes up from dev -> stage -> prod without much thought.